src/Function.ts

```diff
@@ -0,0 +1,131 @@
+import {
+  AllMiddlewareArgs,
+  AnyMiddlewareArgs,
+  Middleware,
+  SlackEventMiddlewareArgs,
+} from './types';
+
+/* Types */
+export interface SlackFunctionExecutedMiddlewareArgs extends SlackEventMiddlewareArgs<'function_executed'> {
+  success: SuccessFn;
+  error: ErrorFn;
```

Comment:
We've never used nouns for passed functions in listener/middleware arguments. For consistency, I prefer using verbs instead. We named similar ones in Steps from Apps as `complete` and `fail`. Reusing these names sound good enough to me.